reference/conanfile/tools/gnu/pkgconfigdeps.rst

```diff
@@ -0,0 +1,83 @@
+.. _conan_tools_pkgconfig:
+
+
+conan.tools.gnu.pkgconfigdeps
+=============================
+
+.. warning::
+
+    These tools are **experimental** and subject to breaking changes.
+
+
+PkgConfigDeps
+-------------
+
+Available since: `1.38.0 <https://github.com/conan-io/conan/releases/tag/1.38.0>`_
+
+
+The ``PkgConfigDeps`` is the dependencies generator for pkg-config. Generates pkg-config files named *<PKG-NAME>.pc*
+(where ``<PKG-NAME`` is the name declared by dependencies in ``cpp_info.name`` or in ``cpp_info.names["pkg_config"]``
```

Comment:
It's copied from the original [PkgConfig generator](https://docs.conan.io/en/latest/reference/generators/pkg_config.html) but I have doubts if I have to add/change something else...

Reply:
You can also add that it can be defined with `pkg_config_name` using `set_property`, (that will be the default way of setting this for 2.0). Please check this: https://docs.conan.io/en/latest/reference/conanfile/attributes.html?highlight=cpp_info%20set_property

Reply:
Actually, for new stuff, we can push as much as we want for new features. If you want to describe ``set_property()`` as the mechanism for this it is fine.  Not necessary to change the PR, it is good so far, but just the idea for moving forward.